ios/VisionCameraTextRecognition.swift

```diff
@@ -40,11 +42,39 @@ public class VisionCameraTextRecognition: FrameProcessorPlugin {
 
     public override func callback(_ frame: Frame, withArguments arguments: [AnyHashable: Any]?) -> Any {
         let buffer = frame.buffer
-        let image = VisionImage(buffer: buffer)
-        image.orientation = getOrientation(orientation: frame.orientation)
-
+        var image: VisionImage?;
         do {
-            let result = try self.textRecognizer.results(in: image)
+            if scanRegion != nil {
```

Comment:
I'm on xcode 15.1 and scanRegion is always nil even after assigned in init. Still figuring it out